### ISSUE TITLE
Not responding at app open

### ISSUE THREAD

**Issue by srilakshmikanthanp** [srilakshmikanthanp/clipbirdroid#2]
After opening the app does not Responding on some phones, Maybe one of the reasons,

- Need to add Loading Screen Since the Application class Creation takes time

**Comment by srilakshmikanthanp** [comment]
Found Actual Problem, Occurring in Android 13

~~~sh
E/AndroidRuntime( 8309): java.lang.SecurityException: Settings key: <bluetooth_name> is only readable to apps with targetSdkVersion lower than or equal to: 31
E/AndroidRuntime( 8309): 	at android.provider.Settings$NameValueCache.getStringForUser(Settings.java:3081)
E/AndroidRuntime( 8309): 	at android.provider.Settings$Secure.getStringForUser(Settings.java:6174)
E/AndroidRuntime( 8309): 	at android.provider.Settings$Secure.getString(Settings.java:6130)
E/AndroidRuntime( 8309): 	at com.srilakshmikanthanp.clipbirdroid.constant.ConstantsKt.appMdnsServiceName(Constants.kt:76)
E/AndroidRuntime( 8309): 	at com.srilakshmikanthanp.clipbirdroid.utility.functions.SslcertKt.generateX509Certificate(sslcert.kt:36)
E/AndroidRuntime( 8309): 	at com.srilakshmikanthanp.clipbirdroid.utility.functions.SslcertKt.generateX509Certificate$default(sslcert.kt:30)
E/AndroidRuntime( 8309): 	at com.srilakshmikanthanp.clipbirdroid.ui.gui.Clipbird.getNewSslConfig(Clipbird.kt:20)
E/AndroidRuntime( 8309): 	at com.srilakshmikanthanp.clipbirdroid.ui.gui.Clipbird.getSslConfig(Clipbird.kt:53)
E/AndroidRuntime( 8309): 	at com.srilakshmikanthanp.clipbirdroid.ui.gui.Clipbird.initialize(Clipbird.kt:88)
E/AndroidRuntime( 8309): 	at com.srilakshmikanthanp.clipbirdroid.ui.gui.service.ClipbirdService$Companion.start(ClipbirdService.kt:156)
E/AndroidRuntime( 8309): 	at com.srilakshmikanthanp.clipbirdroid.ui.gui.SplashActivity.onStart(SplashActivity.kt:25)
E/AndroidRuntime( 8309): 	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1543)
E/AndroidRuntime( 8309): 	at android.app.Activity.performStart(Activity.java:8330)
E/AndroidRuntime( 8309): 	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3670)
E/AndroidRuntime( 8309): 	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
E/AndroidRuntime( 8309): 	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
E/AndroidRuntime( 8309): 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)
E/AndroidRuntime( 8309): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
E/AndroidRuntime( 8309): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2307)
E/AndroidRuntime( 8309): 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/AndroidRuntime( 8309): 	at android.os.Looper.loopOnce(Looper.java:201)
E/AndroidRuntime( 8309): 	at android.os.Looper.loop(Looper.java:288)
E/AndroidRuntime( 8309): 	at android.app.ActivityThread.main(ActivityThread.java:7872)
E/AndroidRuntime( 8309): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime( 8309): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
E/AndroidRuntime( 8309): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
~~~

**Comment by srilakshmikanthanp** [comment]
Fixed with tag v1.1.0
